### PR TITLE
tera-mavr-cv-34: add adaptive design and fix size blocks and added a …

### PR DESCRIPTION
**Pull request by TarasMavrovych**
…few tags

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -298,3 +310,90 @@ ul.my_work_do li {
 
   letter-spacing: 2px;
 }
+
+@media (max-width: 770px) {
```
Comment (PavloSukhinin) [inline assets/css/style.css]: [common breakpoints](https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/)

Comment (vinnikukr) [inline assets/css/style.css]: Why 770px? Look at the task on Trello

File: index.html [hunk 2]
```diff
@@ -24,99 +24,110 @@
...
+        <!-- skills -->
+        <div class="my_sidebar_sections skills">
+          <h2 class="helpers_titles">Навички</h2>
+
+          <h3 class="helpers_sub_titles">Професійні</h3>
+          <ul class="some_list">
+            <li class="tag_list">Дослідження</li>
+            <li class="tag_list">Аналітика</li>
+            <li class="tag_list">Розвиток виробництва</li>
+            <li class="tag_list">Якісність продукції</li>
```
Comment (vinnikukr) [inline index.html]: A lot of repeats: class="tag_list"
You can remove this class and make it easier

File: index.html [hunk 3]
```diff
@@ -24,99 +24,110 @@
...
-          <li class="tag_list">Технік</li>
-        </ul>
-      </div>
-      <!-- /skills -->
+          <h3 class="helpers_sub_titles">Персональні</h3>
+          <ul class="some_list">
+            <li class="tag_list">Критичне та аналітичне мислення</li>
+            <li class="tag_list">Творчі рішення</li>
+            <li class="tag_list">Вирішення проблем</li>
+            <li class="tag_list">Технік</li>
```
Comment (vinnikukr) [inline index.html]: FYI

File: index.html [hunk 4]
```diff
@@ -24,99 +24,110 @@
...
 
-        <ul class="some_list">
-          <li class="tag_list">Українська</li>
-          <li class="tag_list">English A2</li>
-          <li class="tag_list">російська</li>
-        </ul>
+          <ul class="some_list">
+            <li class="tag_list">Українська</li>
+            <li class="tag_list">English A2</li>
+            <li class="tag_list">російська</li>
```
Comment (vinnikukr) [inline index.html]: FYI

File: index.html [hunk 5]
```diff
@@ -24,99 +24,110 @@
...
 
-        <ul class="some_list">
-          <li class="tag_list">Спорт</li>
-          <li class="tag_list">Інтерактивні ігри</li>
-          <li class="tag_list">Читання книжок</li>
-        </ul>
+          <ul class="some_list">
+            <li class="tag_list">Спорт</li>
+            <li class="tag_list">Інтерактивні ігри</li>
+            <li class="tag_list">Читання книжок</li>
```
Comment (vinnikukr) [inline index.html]: FYI

File: index.html [hunk 6]
```diff
@@ -24,99 +24,110 @@
...
+        <div class="my_sidebar_sections dream">
+          <h2 class="helpers_titles">Хочу</h2>
 
-        <ul class="some_list">
-          <li class="tag_list">Стати людиною що зможе усе</li>
-          <li class="tag_list">Познайомитись з Ілоном Маском</li>
-        </ul>
+          <ul class="some_list">
+            <li class="tag_list">Стати людиною що зможе усе</li>
+            <li class="tag_list">Познайомитись з Ілоном Маском</li>
```
Comment (vinnikukr) [inline index.html]: FYI

File: assets/css/style.css [hunk 7]
```diff
@@ -298,3 +310,90 @@ ul.my_work_do li {
 
   letter-spacing: 2px;
 }
+
+@media (max-width: 768px) {
+  .main_section {
+    display: block;
+
+    box-shadow: rgba(149, 157, 165, 0.2) 1px 2px 10px 10px;
+    border-radius: 2%;
```
Comment (PavloSukhinin) [inline assets/css/style.css:319]: do we need this on tablet?

**Review by PavloSukhinin — APPROVED** [review]

File: assets/css/style.css (hunk 7 above)
Comment (TarasMavrovych) [reply assets/css/style.css:319]: looks good to me

**Pull request merged**